src/main/java/com/google/sps/data/Game.java

```diff
@@ -125,8 +125,8 @@ public Builder setNumTimesFinished(int val) {
         }
 
         /**
-        * Sets the number of star votes this game has received.
-        * @param val an int representing the number of times this game has been voted on.
+        * Sets the number of times this game's number of stars has been voted on.
```

Comment:
Try reading this out loud, it sounds off.

Reply:
Reworded as "the number of times the game has received star votes", thanks.